packages/gatsby-plugin-theme-ui/package.json

```diff
@@ -4,6 +4,15 @@
   "main": "index.js",
   "author": "Brent Jackson",
   "license": "MIT",
+  "files": [
+    "src/**/*.js",
+    "/*.js"
+  ],
```

Comment:
This is a pretty unconventional way to use TS, but I'm not sure if it's worse than copying package.json and README to dist and modifying lerna.json.

https://github.com/lerna/lerna/issues/1282